api/utils/emails.js

```diff
@@ -1,11 +1,69 @@
 const Notification = require('../models/Notification');
 const Token = require('../models/Token');
-
 const crypto = require('crypto');
 const sendgrid = require('@sendgrid/mail');
+const fs = require('fs');
+const path = require('path');
 
 sendgrid.setApiKey(process.env.SENDGRID_API_KEY);
 
+const emailDatabaseBackup = (filename) => {
+  try {
+    let attachment = fs
```

Comment:
A few questions here:
- Can this be `const` instead of `let`?
- Why use `readFileSync` instead of `readFile`?
- We use `path.join` elsewhere, can this be changed for consistency?
- I guess the `toString('base64')` is required by SendGrid, right?

Reply:
1. Replaced with const
2. You're right, it should be readFile. Changed that already.
3. Done
4. Yes. The attachment should be encoded to base64. I updated the code for this so it's now more readble.